server/middleware/error-handler.js

```diff
@@ -0,0 +1,15 @@
+var expressErrorHandler = require('errorhandler');
```

Comment:
One more thing to address. `errorhandler` is a `devDependency`, thus this line will thrown when loopback is installed in a project that does not have `errorhandler` in project's dependencies.

Please make `errorhandler` a regular dependency in loopback's `package.json`